doc/src/stdlib/collections.md

```diff
@@ -133,6 +133,9 @@ Base.collect(::Type, ::Any)
 Base.issubset(::Any, ::Any)
 Base.filter
 Base.filter!
+# this also includes the docstring for replace(A, old, new, n)
```

Comment:
Outdated comment. And how it is possible that docstrings with a different signature are included?